### ISSUE TITLE
read KIC files

### ISSUE THREAD

**Issue by davidwhogg** [davidwhogg/HoneyComb#3]
we need to be able to read a set of files with PDC flux from some KIC and construct the footprint object that is needed to run nyquist.py

**Comment by RuthAngus** [comment]
Working on this - need to make timing function more general.

**Comment by davidwhogg** [comment]
Yes for SURE.  Right now the code is just a hack.  When do you get here on Wednesday?  I feel like we should talk at the whiteboard for a bit about all this

**Comment by RuthAngus** [comment]
I'm arriving late Tuesday night, so I'll be there in the morning on Wednesday!

**Comment by davidwhogg** [comment]
Excellent.  Let's start at 9, or even 08:30, if you can get to Third Rail by then.
